### ISSUE TITLE
Feature request:Set system default repository

### ISSUE THREAD

**Issue by pjw91** [nicolargo/ubuntupostinstall#11]
The `sources.list` of a fresh-installed Ubuntu is using the host of "http://archive.ubuntu.com", which is really slow for some country.

And the default "Server for XXX" is sometimes not the best of all servers.
For example, for Taiwan, the `free.NCHC.org` mirror is the fastest, but the `mirror://` protocal doesn't think so.

So please add a secion/option like `default_ppa` to replace the default repo into the specified one.

In other words,
automatically change `/etc/apt/sources.list`

```
deb http://archive.ubuntu.com/ubuntu precise-proposed restricted main multiverse universe
...
```

to

```
deb http://free.nchc.org.tw/ubuntu/ precise-proposed restricted main multiverse universe
...
```
